test/test-rotatesTetrominos.js

```diff
@@ -0,0 +1,32 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris');
+describe("rotate L-tetromino", function (){
+    class Tetramino = {
```

Comment:
This should not be part of test

Reply:
How I will be test?

Reply:
How do you test methods? 
You put module name in front and a dot